.github/workflows/trybot.yml

```diff
@@ -38,6 +43,9 @@ jobs:
         working-directory: ./play
       - name: Test
         run: go test ./...
+      - name: Test play
+        run: go test ./...
+        working-directory: ./play
```

Comment:
I notice that this is added. Just checking that this is on purpose, and I wonder if it was harmful that it wasn't there before.

Reply:
There are no tests in play, there might be some in the future. Adding this really to reduce the diff. There is zero impact because there are no tests.